docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
+
+volumes:
+  csgo-data:
+    name: csgo-data
```

Comment:
explicitly specifying the `name` prevents docker-compose from prefixing the volume name with the container name. This makes the data volume reusable outside of docker-compose context.